cmd/launcher/main.go

```diff
@@ -23,6 +23,12 @@ func main() {
 	// re-leveled during options parsing.
 	logger := logutil.NewServerLogger(env.Bool("LAUNCHER_DEBUG", false))
 
+	level.Info(logger).Log(
```

Comment:
This also runs in launcher.go correct? Not saying it's wrong to have it here, but it does result in some duplicate info in the logs. Perhaps we could remove it from `launcher.go`?

Reply:
I think they're slightly different. At least, there's a bunch of other stuff that might happen in between them. See also, tracking down that boltdb hang